CountingSort/app_resources/prefix_sum_shader.comp.hlsl

```diff
@@ -42,12 +39,20 @@ uint32_t3 nbl::hlsl::glsl::gl_WorkGroupSize()
 [numthreads(WorkgroupSize,1,1)]
 void main(uint32_t3 ID : SV_GroupThreadID, uint32_t3 GroupID : SV_GroupID)
 {
-    nbl::hlsl::sort::counting <PtrAccessor, PtrAccessor, PtrAccessor> counter;
-    PtrAccessor input_accessor = PtrAccessor::createAccessor(pushData.inputKeyAddress);
-    PtrAccessor scratch_accessor = PtrAccessor::createAccessor(pushData.scratchAddress);
+    nbl::hlsl::sort::CountingParameters < uint32_t > params;
+    params.dataElementCount = pushData.dataElementCount;
+    params.elementsPerWT = pushData.elementsPerWT;
+    params.minimum = pushData.minimum;
+    params.maximum = pushData.maximum;
+
+    nbl::hlsl::sort::counting <WorkgroupSize, BucketCount, uint32_t, PtrAccessor, PtrAccessor, PtrAccessor, SharedAccessor> counter;
+    PtrAccessor input_accessor = PtrAccessor::create(pushData.inputKeyAddress);
+    PtrAccessor scratch_accessor = PtrAccessor::create(pushData.scratchAddress);
```

Comment:
replace occurences of `scratch` in naming with `histogram`

Reply:
Done